### PR TITLE
Incrementalified.

### PR DESCRIPTION
**Pull request by casz**
awaiting https://github.com/jenkins-infra/repository-permissions-updater/pull/687

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -10,11 +10,13 @@
...
   <artifactId>accurev</artifactId>
   <packaging>hpi</packaging>
-  <version>0.7.17-SNAPSHOT</version>
+  <version>${revision}${changelist}</version>
   <name>Jenkins AccuRev plugin</name>
   <description>This plugin allows you to use AccuRev as a SCM.</description>
   <url>http://wiki.jenkins-ci.org/display/JENKINS/AccuRev+Plugin</url>
   <properties>
+        <revision>0.7.17</revision>
+        <changelist>-SNAPSHOT</changelist>
```
Comment (jglick) [inline pom.xml]: BTW [reindent](https://github.com/jenkinsci/incrementals-tools/blob/bc7488dd22fb2ea8813d82ec3ff5caf8dd762bb8/maven-plugin/src/main/java/io/jenkins/tools/incrementals/maven/IncrementalifyMojo.java#L142)

Comment (casz) [reply pom.xml]: I used `mvn incrementals:incrementalify` to increment that accidentally have misindendet I guess so I blame you 😝

Comment (jglick) [reply pom.xml]: Yes, hence the link!

**Comment by jglick** [comment]
Try fixing the indentation to trigger a new build—ought to get auto-deployed this time.

File: pom.xml (hunk 1 above)
Comment (casz) [reply pom.xml]: I missed the link, sorry 😢

**Pull request merged**